xbmc/cores/paplayer/VideoPlayerCodec.cpp

```diff
@@ -213,7 +213,7 @@ bool VideoPlayerCodec::Init(const std::string &strFile, unsigned int filecache)
   m_bCanSeek = false;
   if (m_pInputStream->Seek(0, SEEK_POSSIBLE))
   {
-    if (Seek(1) != DVD_NOPTS_VALUE)
+    if (Seek(1) != (int64_t) DVD_NOPTS_VALUE)
     {
```

Comment:
@FernetMenta after your redefinition we have two definition anyways - the other one in libXBMC_pvr.h which still has the old value.